pages/glossary/decorative.md

```diff
@@ -3,4 +3,4 @@ title: Decorative
 key: decorative
 ---
 
-Serving only an aesthetic purpose, providing no information, and having no functionality.
\ No newline at end of file
+Serving only an aesthetic purpose, providing no information, and having no functionality. An `img` element can be marked as decorative, by using either `role="presentation"`, `role="none"` or an empty alt attribute `alt=""`.
```

Comment:
I think this definition contradicts itself. An img can have a purpose, even when it's marked up with role=presentation. It shouldn't, it's an accessibility failure, but it can. I don't think you need this "decorative" definition in this rule at all. Why did you add this?

Reply:
@WilcoFiers I have updated the definition to separate out the implementation detail as a note after discussion with Anne and Jey

Reply:
https://github.com/auto-wcag/auto-wcag/pull/263#discussion_r249721940